.bazelrc

```diff
@@ -0,0 +1,13 @@
+# Bazel settings that apply to this repository.
+# Take care to document any settings that you expect users to apply.
+# Settings that apply only to CI are in .github/workflows/ci.bazelrc
```

Comment:
Yes, I've used that `--config=ci` scheme in the past as well. The main reason to make another file is because in a complex CI setup, you end up calling `bazel` in lots of different places. You need to query things to discover what to build & test, someone's code requires a different configuration (test fails without `-c opt` for example).
It's easy to accidentally forget the `--config=ci` flag in one of the many invocations of bazel, or especially hard to ensure that later maintainers of the CI scripts don't omit it.

So, the pattern I usually use is to copy the ci.bazelrc file into the system or user's location at the beginning of the pipeline. This way the CI rc settings are always applied without needing to remember what flags to pass each bazel call.

To be fair, in this repo, I don't think it makes much difference which we pick. If we want to simplify things, I wouldn't even bother with a separate rc file, just use the `ci` config in the same `.bazelrc` with the rest.

Reply:
My preference is to keep all the CI configuration adjacent, more strongly than to keep all the bazelrc's adjacent. I'm open to discussion on it of course.

Reply:
> It's easy to accidentally forget the `--config=ci` flag in one of the many invocations of bazel, or especially hard to ensure that later maintainers of the CI scripts don't omit it.
> 
> So, the pattern I usually use is to copy the ci.bazelrc file into the system or user's location at the beginning of the pipeline. This way the CI rc settings are always applied without needing to remember what flags to pass each bazel call.

IIUC the same can be achieved by using `build:ci` etc. in `.bazelrc` and then writing `user.bazelrc` or `~/.bazelrc` with the following contents at the beginning of the CI pipeline.
```
build --config=ci
query --config=ci
...
```